include/boost/math/tools/luroth_expansion.hpp

```diff
@@ -29,17 +29,17 @@ class luroth_expansion {
             throw std::domain_error("Cannot convert non-finites into a Luroth representation.");
         }
         d_.reserve(50);
-        Real dn = floor(x);
-        d_.push_back(static_cast<Z>(dn));
-        if (dn == x)
+        Real dn1 = floor(x);
+        d_.push_back(static_cast<Z>(dn1));
+        if (dn1 == x)
         {
            d_.shrink_to_fit();
            return;
         }
         // This attempts to follow the notation of:
         // "Khinchine's constant for Luroth Representation", by Sophia Kalpazidou.
-        x = x - dn;
-        Real computed = dn;
+        x = x - dn1;
+        Real computed = dn1;
```

Comment:
I'm confused about what I did on this one that needed the variable name change.

Reply:
`dn` is defined [again here](https://github.com/boostorg/math/blob/63a16971afc470e1772ed99da2ed2a2835a8954c/include/boost/math/tools/luroth_expansion.hpp#L52). Changed the name to suppress shadowing warnings.